graph_weather/models/forecast.py

```diff
@@ -29,6 +30,9 @@ def __init__(
         hidden_layers_decoder: int = 2,
         norm_type: str = "LayerNorm",
         use_checkpointing: bool = False,
+        # New constraint parameters
+        constraint_type: str = "additive",  # "additive", "multiplicative", or "softmax"
+        apply_constraints: bool = True,
```

Comment:
Maybe combining these two parameters make more sense, and default to "none"/`None`
```suggestion
        constraint_type: str = "additive",  # "additive", "multiplicative", "softmax", or "none"
```

Reply:
Ack, made the change